build/hpc_install.sh

```diff
@@ -0,0 +1,157 @@
+# Cluster specific setup
+if [[ $1 == "longleaf" ]]; then
+    # Setup general purpose user variables needed for Longleaf
+    USERO=$( echo $USER | awk '{ print substr($0, 1, 1) }' )
+    USERN=$( echo $USER | awk '{ print substr($0, 2, 1) }' )
+    USERDIR="/users/$USERO/$USERN/$USER"
+    cd $USERDIR
+
+    # Load required modules
+    module purge
+    module load gcc/9.1.0
+    module load anaconda/2023.03
+    module load git
+elif [[ $1 == "rockfish" ]]; then
+    # Setup general purspose user variables needed for RockFish
+    USERDIR="/scratch4/struelo1/flepimop-code/$USER/"
+
+    # Load required modules
+    module purge
+    module load gcc/9.3.0
+    module load anaconda/2020.07
+    module load git/2.42.0
+else
+    echo "The cluster name '$1' is not recognized, must be one of: 'longleaf', 'rockfish'."
+    exit 1
+fi
+
+# Make sure the credentials is is where we expect and have the right perms
+if [ ! -f "$USERDIR/slack_credentials.sh" ]; then
+    echo "You need to place sensitive credentials in '$USERDIR/slack_credentials.sh'."
+    exit 1
+fi
+chmod 600 $USERDIR/slack_credentials.sh
+source $USERDIR/slack_credentials.sh
+
+# Ensure we have a $FLEPI_PATH
+if [ -z "${FLEPI_PATH}" ]; then
+    echo "An explicit \$FLEPI_PATH was not provided, setting to '$USERDIR/flepiMoP'."
+    export FLEPI_PATH="$USERDIR/flepiMoP"
+fi
+
+# Test that flepiMoP is located there
+if [ ! -d "$FLEPI_PATH" ]; then
+    echo "Did not find flepiMoP at '$FLEPI_PATH', cloning on your behalf."
+    git clone https://github.com/HopkinsIDD/flepiMoP.git $FLEPI_PATH
+elif [ ! -d "$FLEPI_PATH/.git" ]; then
+    echo "The flepiMoP found at '$FLEPI_PATH' is not a git clone, unsure of how to proceed."
+    exit 1
+fi
+
+# Setup the conda environment
+if [ ! -d "$USERDIR/flepimop-env" ]; then
+cat << EOF > $USERDIR/environment.yml
+channels:
+- conda-forge
+- defaults
+dependencies:
+- python=3.10
+- pip
+- r-base>=4.4
+- r-essentials
+- r-devtools
+- pyarrow=17.0.0
+- r-arrow=17.0.0
+# Manually specify this one because of the paths for libudunits2 on longleaf
+- r-sf 
+# This packages are probably missing from the DESCRIPTION of the R packages
+- r-optparse
+- pip:
+    - git+https://github.com/HopkinsIDD/flepiMoP.git#subdirectory=flepimop/gempyor_pkg
```

Comment:
Most of the time, folks would want to use a specific branch here. Perhaps the install flepiMoP step should be separated from the build flepiMoP environment step ?

Reply:
"Separate branches for install vs build for simulation" ... is that what I'm understanding here? That's very... not-a-library energy. I understand dev is still underway, and so we need to make sure the update element of this works well.

Reply:
yeah, I mean it's a library (you can still run a old config prefectly) but there are also new features constantly needed as our needs increase (there were variant, then age, then seasonality, now races, with harder and harder inference problems and all) so I do think that our scenario modeling hub runs will often require some new features.

Reply:
This now lets you install a consistent set of `gempyor` and R packages from a branch, just checkout the correct branch and then run the install script. Does that satisfy the issue raised here?

Reply:
yes, thanks !